staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -81,8 +81,8 @@ var (
 	kubectlExeRegexp = regexp.MustCompile(`^.*((?i:kubectl\.exe))`)
 )
 
-// Register all metrics.
-func Register() {
+func init() {
```

Comment:
not a fan of importing the metrics package autoregistering

Reply:
Previously, autoregistering by importing `k8s.io/apiserver/pkg/endpoints`

Reply:
Is there an easy way to lazily register when metrics are actually used?

Reply:
Yes, we can implement lazy register by register in `Record` and `Record`. But why we need  `lazily register` ?

Reply:
I agree. pkg/endpoints/metrics is pretty deep in the apiserver. Anybody importing that outside of k8s.io/apiserver?

Reply:
I donot know.  Maybe user private extended apiservers import it.

Reply:
But these are not directly import.